packages/fluentui/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -109,31 +114,46 @@ class AccordionTitle extends UIComponent<WithAsProp<AccordionTitleProps>, any> {
     _.invoke(this.props, 'onFocus', e, this.props);
   };
 
+  handleWrapperOverrides = () => ({
+    onFocus: (e: React.SyntheticEvent) => {
+      this.handleFocus(e);
+    },
+    onClick: (e: React.SyntheticEvent) => {
+      this.handleClick(e);
+    }
+  });
+
   renderComponent({ ElementType, classes, unhandledProps, styles, accessibility }) {
-    const { contentRef, children, content, indicator } = this.props;
+    const { contentRef, children, content, indicator, wrapper } = this.props;
 
     const contentElement = (
       <Ref innerRef={contentRef}>
-        <div
-          onFocus={this.handleFocus}
-          onClick={this.handleClick}
-          className={cx(AccordionTitle.slotClassNames.contentWrapper, classes.contentWrapper)}
-          {...accessibility.attributes.content}
-          {...applyAccessibilityKeyHandlers(accessibility.keyHandlers.content, unhandledProps)}
-        >
-          {Box.create(indicator, {
-            defaultProps: () => ({
-              styles: styles.indicator,
-              accessibility: indicatorBehavior
-            })
-          })}
-          {Box.create(content, {
-            defaultProps: () => ({
-              as: 'span',
-              styles: styles.content
-            })
-          })}
-        </div>
+        {Box.create(wrapper, {
+          defaultProps: () => ({
+            className: cx(AccordionTitle.slotClassNames.contentWrapper, classes.contentWrapper),
```

Comment:
```suggestion
            className: AccordionTitle.slotClassNames.contentWrapper,
            styles: styles.contentWrapper,
```

As it's a `Box` we should use `styles`